tests/unit/dataactvalidator/utils.py

```diff
@@ -12,15 +12,21 @@ def fetch_sql_tpl(filename):
         return f.read()
 
 
-def models_are_valid(rule_file, staging_db, *models):
+def insert_submission(db, submission):
+    db.session.add(submission)
+    db.session.commit()
+    return submission.submission_id
+
+
+def run_sql_rule(rule_file, staging_db, submission_id, *models):
```

Comment:
Separating out submission creation from `run_sql_rule` makes this a bit less self-contained. What do you think of this signature instead?

```python
def run_sql_rule(rule_file, staging_db, submission=None, models=None):
    if submission is None:
        submission = SubmissionCreationHere()
    if models is None:
        models = []
    submision_id = insert_submission(staging_db, submission)
    ...
```